spec/models/category_user_spec.rb

```diff
@@ -269,23 +269,28 @@ def category_ids_at_level(level)
     end
   end
 
-  describe "#indirectly_muted_category_ids" do
+  describe "#muted_category_ids" do
```

Comment:
```suggestion
  describe ".muted_category_ids" do
```

Seems like this is a class method and not an instance method.